### ISSUE TITLE
Notes on first decipherment experiments

### ISSUE THREAD

**Issue by j0ma** [j0ma/ancestral-decipherment#2]
Using PanLex alone is not feasible as the strings are too short.
This will cause the frequency-encoded "ciphertexts" to be awfully short and, as a result, many words get encoded to something like `0 1 2 3 4 5`.
While this might've been a good idea with plain decipherment with tons of data, it's a bit hard to do with just a bilingual dictionary.

Understandably the results were abysmal with SER over 100%.
To fix this, I moved from PanLex to Tatoeba.
Those experiments are still running but seem to exhibit better loss dynamics (no divergence).

**Comment by j0ma** [comment]
**PanLex** overfit (similar romanized or not):

![image](https://user-images.githubusercontent.com/10352674/193665408-6eca1c96-f027-4a75-809a-c343cf04770c.png)

**Tatoeba**:

Hebrew script:
![image](https://user-images.githubusercontent.com/10352674/193665449-ef3d5bd4-dbd8-4917-a941-b8c7dd0fd427.png)

Romanized (using [ibleaman/yiddish](https://github.com/ibleaman/yiddish))

![image](https://user-images.githubusercontent.com/10352674/193665488-45b10acf-29da-460e-8ceb-a078a5de6ddd.png)

**Comment by j0ma** [comment]
Given that using dictionaries alone seems infeasible, would need **parallel** data for this? 

Or maybe build "fake sentences" by simply concatenating individual words? That way no need to hunt for parallel data.

**Comment by j0ma** [comment]
Both models trained for 20 epochs using 1xV100 GPU with `update_freq=4`, i.e. training with delayed updates simulating 4xGPUs. 

# Transformer model from paper

```
SER	Accuracy	Language
0.163	27.983	multi
```

# Halved learning rate

Halving the learning rate seems to bring gains

```
SER	Accuracy	Language
0.139	31.326	multi
```
